src/qt/bitcoin.cpp

```diff
@@ -573,7 +573,7 @@ int main(int argc, char *argv[])
     {
         HelpMessageDialog help(NULL, mapArgs.count("-version"));
         help.showOrPrint();
-        return 1;
+        return 0;
```

Comment:
Technically, yes, but there are few other `return`s/`exit`s in `main()` in src/qt/bitcoin.cpp (not to mention other places) and originally I tried to be consistent with the surrounding code, just fixing some parts to return correct value. Shall we replace all such magic numbers (here?) with constants then?